### PR TITLE
FT-01: Creating App and Adding Common Layout

### PR DESCRIPTION
**Pull request by Jeysh-git**
In this PR:
 - added plugin ember-bootstrap@2.6.1
 - added plugin ember-cli-sass@10.0.0
 - added plugin ember-svg-jar@2.3.3
 - added common main left navigation bar, header, footer, images.
 
output screenshot:
Screenshot 2022-09-16 at 11 39 30 AM

Figma-link: https://www.figma.com/file/i9whcNdWDHjHsXVdpoYyFh/New-Joinee?node-id=0%3A1

### CODE REVIEW

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+    <div class="header">
+        <img class = "headerimg" src ="images/brand-header-logo.png" width="34px" height="34px">
+        <div class="header-text">Talent Center</div>
+    </div>
+    
+    <div class="footer">
+        <div class="footer-text">HR Software by Freshworks   |    Knowledge base
+        </div>
+    </div>
+{{outlet}}
```
Comment (Naomi-Mathews) [inline app/templates/application.hbs]: outlet should be before footer

File: app/styles/app.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+.navbar-left {
+    height: 820px;
+    width: 60px;
+    position: absolute;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    right:0;
+    bottom:0;
+    background-color: $leftnav-color;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss:18]: why is left, right , bottom and top required?

File: app/styles/app.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    overflow: auto;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: Why is overflow auto required? Are we expecting scroll?

File: app/styles/app.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+ }
+
+ .leftbar-logo {
+    position: absolute;
+    height: 52px;
+    left: 0%;
+    right: 0%;
+    top: 0px;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: why is left, right , bottom and top required? Also when a value is 0 no need to specify % or px

File: app/styles/app.scss [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    width:60px;
+    background: $leftnav-color;
+    box-shadow: 0px 1px 0px #41637C, 0px 2px 0px #27303E;
+ }
+
+ .logoimg {
+    position: absolute;
+    left: 30%;
+    right: 28.33%;
+    top: 14px;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: why is left and right required?

File: app/styles/app.scss [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    width:1380px;
+    height:54px;
+    background-color: #ffffff;
+    margin-left:60px;
+    display:flex;
+    justify-content: flex-start;
+ }
+
+ .headerimg{
+    position: absolute;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: why is position absolute required here? Acheive it using flex

File: app/styles/app.scss [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    background: #FFFFFF;
+    width:34px;
+    height:34px;
+    top:8.5px;
+    left:34.63px;
+ }
+
+ .header-text{
+    position: absolute;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: position absolute not required here

File: app/styles/app.scss [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+ }
+
+ .footer {
+    box-sizing: border-box;
+    position: absolute;
+    width: 1380px;
+    height: 20px;
+    left: 60px;
+    top: 800px;
+    bottom:0%;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: top and bottom properues not required

File: app/styles/app.scss [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+    background: #F5F7F9;
+    border: 1px solid #CFD7DF;
+ }
+
+ .footer-text{
+    width:280px;
+    height:16px;
+    left:530px;
+    position: absolute;
```
Comment (Naomi-Mathews) [inline app/styles/app.scss]: why is position absolute required for this?

**Review by Naomi-Mathews — CHANGES_REQUESTED** [review]

File: app/templates/application.hbs (hunk 1 above)
Comment (Jeysh-git) [reply app/templates/application.hbs]: Updated.

File: app/styles/app.scss (hunk 2 above)
Comment (Jeysh-git) [reply app/styles/app.scss:18]: added appropriate styles.

File: app/styles/app.scss (hunk 3 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: removed overflow

File: app/styles/app.scss (hunk 4 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: Done

File: app/styles/app.scss (hunk 5 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: added appropriate styles

File: app/styles/app.scss (hunk 6 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: added flex.

File: app/styles/app.scss (hunk 7 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: removed.

File: app/styles/app.scss (hunk 8 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: Done.

File: app/styles/app.scss (hunk 9 above)
Comment (Jeysh-git) [reply app/styles/app.scss]: added appropriate styles

**Review by Naomi-Mathews — APPROVED** [review]

**Pull request merged**